app/views/suggestions/_preview.html.erb

```diff
@@ -7,9 +7,9 @@
       </div>
       <div class='media-body'>
         <h4 class='media-heading text-default'><%= suggestion.title %></h4>
-        <p class='media-description'><%= truncate(suggestion.description, length: 250) %></p>
+        <p class='media-description'><%= truncate(suggestion.description.html_safe, length: 250) %></p>
```

Comment:
Yeah looks like we were already doing this in the other `_preview` files with `truncate_html` – here's an example: `<p class='media-description'><%= truncate_html(idea.description, length: 250).html_safe %></p>`